archaius2-api/src/main/java/com/netflix/archaius/api/ArchaiusType.java

```diff
@@ -81,6 +82,14 @@ public Type getRawType() {
         return rawType;
     }
 
+    public boolean isMap() {
```

Comment:
Maybe move these two to be private static helper methods in AbstractConfig? Then things would work with any `ParametrizedType`, not just with `ArchaiusType`.

Reply:
Good call, moved these into the AbstractConfig instead